plugins/reachability/src/main/java/org/qbicc/plugin/reachability/ReachabilityInfo.java

```diff
@@ -133,10 +134,19 @@ public static void forceCoreClassesReachable(CompilationContext ctxt) {
         info.analysis.processInstantiatedClass(unsafe, true, false,null);
         info.analysis.processClassInitialization(unsafe);
 
-        // Hack around the way NoGC entrypoints are registered and then not used until LOWERING PHASE...
-        LOGGER.debugf("Forcing org.qbicc.runtime.gc.nogc.NoGcHelpers reachable");
-        LoadedTypeDefinition nogc = ctxt.getBootstrapClassContext().findDefinedType("org/qbicc/runtime/gc/nogc/NoGcHelpers").load();
-        info.analysis.processClassInitialization(nogc);
+        LOGGER.debugf("Forcing all runtime functions reachable/initialized");
+        ctxt.foreachRuntimeFunction(elem -> {
+            LoadedTypeDefinition ltd = elem.getEnclosingType().load();
+            info.analysis.processClassInitialization(ltd);
+            if (elem instanceof MethodElement me) {
+                if (me.isStatic()) {
+                    info.analysis.processReachableStaticInvoke(me, null);
+                } else {
+                    info.analysis.processReachableInstanceMethodInvoke(me, null);
+                }
+            }
+        });
+        LOGGER.debugf("Completed processing of runtime functions");
```

Comment:
With the above change, I would have the Driver do this work to pre-seed the queue for each phase.

Reply:
That makes sense.  May need to add a method to ReachabilityInfo that the Driver can invoke, but that's cleaner than exposing a foreachRuntimeFunction on the CompilationContext.

Reply:
Done.  Didn't know about ElementHandlers before.  Nice.